lib/node_modules/@stdlib/stats/base/dstdevtk/src/addon.c

```diff
@@ -35,11 +35,29 @@
 static napi_value addon( napi_env env, napi_callback_info info ) {
 	STDLIB_NAPI_ARGV( env, info, argv, argc, 4 );
 	STDLIB_NAPI_ARGV_INT64( env, N, argv, 0 );
-	STDLIB_NAPI_ARGV_INT64( env, stride, argv, 3 );
 	STDLIB_NAPI_ARGV_DOUBLE( env, correction, argv, 1 );
-	STDLIB_NAPI_ARGV_STRIDED_FLOAT64ARRAY( env, X, N, stride, argv, 2 );
-	STDLIB_NAPI_CREATE_DOUBLE( env, stdlib_strided_dstdevtk( N, correction, X, stride ), v );
+	STDLIB_NAPI_ARGV_INT64( env, strideX, argv, 3 );
+	STDLIB_NAPI_ARGV_STRIDED_FLOAT64ARRAY( env, X, N, strideX, argv, 2 )
+	STDLIB_NAPI_CREATE_DOUBLE( env, stdlib_strided_dstdevtk( N, correction, X, strideX ), v );
```

Comment:
```suggestion
	STDLIB_NAPI_CREATE_DOUBLE( env, API_SUFFIX(stdlib_strided_dstdevtk)( N, correction, X, strideX ), v );
```

@aayush0325 I think I've been missing this change in various PRs, but we need to wrap in `API_SUFFIX`.